smslibrary/src/main/java/com/eis/smslibrary/SMSReceivedBroadcastReceiver.java

```diff
@@ -1,25 +1,44 @@
+/*
+ * Copyright (C) 2015 The Android Open Source Project
+ * Modifications Copyright (C) 2019 Giovanni Velludo
+ * Not a Contribution, Apache license notifications and license are retained
+ * for attribution purposes only
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
What's up with this license, just put here?

Reply:
I copied and slightly modified three methods from AOSP's default Messaging app, to handle reception of multi-part messages properly. Because of this I also had to copy this notice, to comply with the Apache License 2.0.